benchmark/Streamly/Benchmark/Data/Parser/ParserD.hs

```diff
@@ -32,6 +32,7 @@ import qualified Streamly.Internal.Data.Parser.ParserD as PR
 import qualified Streamly.Internal.Data.Producer as Producer
 import qualified Streamly.Internal.Data.Producer.Source as Source
 import qualified Streamly.Internal.Data.Stream.IsStream as IP
+import qualified Streamly.Internal.Data.Stream.StreamD.Generate as SD
```

Comment:
We usually import it `as D`. Also you can import `StreamD` itself (which is more stable wrt code changes) rather than importing the lower level `StreamD.Generate`.